### PR TITLE
ref(controller): rename folder to spot-controller

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
Force pushed to fix conflicts with the root README.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ This is the mono-repo for jitsi meet's room integration, named Spot.
 
 in-room-controller/ contains the Android and iOS clients to be used as a permanent in-room remote
 spot-admin/ contains a mock server for testing integration between Spot and a backend service
+spot-controller/ contains the Android and iOS clients to be used as a permanent in-room remote
```
Comment (zbettenbuk) [inline README.md]: Ami I right assuming that this will also be the standalone store-version of the remote, so then not only the in-room controller?

Comment (virtuacoplenny) [reply README.md]: I didn't know there was some other intent of the react-native app. I did a bad conflict fix for this so I have to change it anyway. I'll remove the in-room part.

Comment (paweldomas) [reply README.md]: should the description at line 5 for `in-room-controller/` directory be removed  ?

Comment (virtuacoplenny) [reply README.md]: Yes, that was the bad conflict fix I was talking about.

**Pull request merged**